components/knative-function-controller/.gitignore

```diff
@@ -0,0 +1,25 @@
+
```

Comment:
shouldn't we enter regex in the .gitignore at root level?
something like **/knative-function-controller/*.exe,  **/knative-function-controller/*.out, etc

Reply:
That make sense but it is not common in Kyma as every project has their own .gitignore file.